src/org/loklak/objects/Timeline2.java

```diff
@@ -0,0 +1,457 @@
+/**
```

Comment:
@vibhcool: Could you please point out the things in this class that are different from `Timeline`?